tests/test_chromatic.py

```diff
@@ -164,7 +164,7 @@ def test_draw_add_commutativity():
     chromatic_kimage = galsim.ImageCD(stamp_size, stamp_size, scale=pixel_scale)
     # use chromatic parent class to draw without ChromaticConvolution acceleration...
     t4 = time.time()
-    integrator = galsim.integ.ContinuousIntegrator(galsim.integ.midpt, N=N, use_endpoints=False)
+    integrator = galsim.integ.ContinuousIntegrator(galsim.integ.midptRule, N=N, use_endpoints=False)
```

Comment:
I think I'd rather this be merely a deprecation at this point, rather than a direct API change.  I don't think there is a need to break backwards compatibility here.

It seems like this could be done in the constructors for `SampleIntegrator` and `ContinuousIntegrator`. 
 You could add an explicit check for `midpt`, `trapz` or `np.trapz` (which was also previously documented as allowed), and switch the `rule` over to the corresponding new function with a deprecation warning.